colored_pointcloud_node:
  ros__parameters:
    rgb_image_topic: "stereo/left/rgb"
    depth_image_topic: "stereo/left/depth"
    rgb_info_topic: "stereo/left/camera_info"
    depth_info_topic: "stereo/left/camera_info"

orb_slam3_pointcloud_stitcher:
  ros__parameters:
    depth_pointcloud_topic: "/camera/colored_pointcloud"
    input_pointcloud_rate: 10.0
    robot_base_frame: "base_footprint" # should be the same frame as the one used in the slam